checker/bin/check_md5sum

```diff
@@ -6,39 +6,41 @@ import json
 import sys
 import os
 
+
 def main():
-  parser = argparse.ArgumentParser(description='Checks to see if a given file contains a certain string and then returns a JSON.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-  parser.add_argument('--input-file', type=str, help='file input')
-  parser.add_argument('--md5', type=str, help='expected md5')
-  args = parser.parse_args()
+    parser = argparse.ArgumentParser(description='Checks to see if a given file contains a certain string and then returns a JSON.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--input-file', type=str, help='file input')
+    parser.add_argument('--md5', type=str, help='expected md5')
+    args = parser.parse_args()
+
+    result_code = check_file(args.input_file, args.md5)
 
-  result = check_file(args.input_file, args.md5)
+    results_file = open("results.json", "w")
+    if result_code == 0:
+        results_file.write('{ "overall" : true, "steps" : { "md5sum_check" : true } }')
+    else:
+        results_file.write('{ "overall" : false, "steps" : { "md5sum_check" : false } }')
+    results_file.close()
 
-  results_file  = open("results.json", "w")
-  if (result == 0):
-    results_file.write('{ "overall" : true, "steps" : { "md5sum_check" : true } }')
-  else:
-    results_file.write('{ "overall" : false, "steps" : { "md5sum_check" : false } }')
-  results_file.close()
+    log_file = open("log.txt", "w")
+    log_file.write("")
+    log_file.close()
 
-  log_file  = open("log.txt", "w")
-  log_file.write("")
-  log_file.close()
+    sys.exit(result_code)
```

Comment:
As we discussed, I think this is the only line that matters for the fix in this file. In a perfect world, the PR would probably only have this line, and the formatting would done in a separate PR... But not worth changing.